{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Could Not Find C:\\Users\\mkcarra\\Anaconda3\\Notebooks\\Euler*.nbconvert.ipynb\n"
     ]
    }
   ],
   "source": [
    "!del Euler*.nbconvert.ipynb"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Euler1.ipynb', 'Euler10.ipynb', 'Euler11.ipynb', 'Euler12.ipynb', 'Euler13.ipynb', 'Euler14.ipynb', 'Euler15.ipynb', 'Euler16.ipynb', 'Euler2.ipynb', 'Euler20.ipynb', 'Euler3.ipynb', 'Euler4.ipynb', 'Euler5.ipynb', 'Euler6.ipynb', 'Euler7.ipynb', 'Euler8.ipynb', 'Euler81.ipynb', 'Euler9.ipynb', 'Euler96.ipynb']\n"
     ]
    }
   ],
   "source": [
    "filelist = !dir /B Euler*ipynb\n",
    "print(filelist)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[NbConvertApp] Converting notebook Euler1.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 818 bytes to Euler1.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler10.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] ERROR | Timeout waiting for execute reply (30s).\n",
      "                If your cell should take longer than this, you can increase the timeout with:\n",
      "\n",
      "                    c.ExecutePreprocessor.timeout = SECONDS\n",
      "\n",
      "                in jupyter_nbconvert_config.py\n",
      "                \n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 189, in run_cell\n",
      "    msg = self.kc.shell_channel.get_msg(timeout=timeout)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\jupyter_client\\blocking\\channels.py\", line 57, in get_msg\n",
      "    raise Empty\n",
      "queue.Empty\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\Scripts\\jupyter-nbconvert-script.py\", line 5, in <module>\n",
      "    main()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\jupyter_core\\application.py\", line 267, in launch_instance\n",
      "    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\traitlets\\config\\application.py\", line 596, in launch_instance\n",
      "    app.start()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 293, in start\n",
      "    self.convert_notebooks()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 457, in convert_notebooks\n",
      "    self.convert_single_notebook(notebook_filename)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 428, in convert_single_notebook\n",
      "    output, resources = self.export_single_notebook(notebook_filename, resources, input_buffer=input_buffer)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 357, in export_single_notebook\n",
      "    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 165, in from_filename\n",
      "    return self.from_file(f, resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 183, in from_file\n",
      "    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\notebook.py\", line 26, in from_notebook_node\n",
      "    nb_copy, resources = super(NotebookExporter, self).from_notebook_node(nb, resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 130, in from_notebook_node\n",
      "    nb_copy, resources = self._preprocess(nb_copy, resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 302, in _preprocess\n",
      "    nbc, resc = preprocessor(nbc, resc)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\base.py\", line 47, in __call__\n",
      "    return self.preprocess(nb,resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 145, in preprocess\n",
      "    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\base.py\", line 70, in preprocess\n",
      "    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 161, in preprocess_cell\n",
      "    outputs = self.run_cell(cell)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 207, in run_cell\n",
      "    raise exception(\"Cell execution timed out, see log\"\n",
      "TimeoutError: Cell execution timed out, see log for details.\n",
      "[NbConvertApp] Converting notebook Euler11.ipynb to notebook\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 14, in parse_json\n",
      "    nb_dict = json.loads(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\__init__.py\", line 319, in loads\n",
      "    return _default_decoder.decode(s)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\decoder.py\", line 339, in decode\n",
      "    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\decoder.py\", line 355, in raw_decode\n",
      "    obj, end = self.scan_once(s, idx)\n",
      "json.decoder.JSONDecodeError: Expecting ',' delimiter: line 93 column 1 (char 4470)\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\Scripts\\jupyter-nbconvert-script.py\", line 5, in <module>\n",
      "    main()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\jupyter_core\\application.py\", line 267, in launch_instance\n",
      "    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\traitlets\\config\\application.py\", line 596, in launch_instance\n",
      "    app.start()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 293, in start\n",
      "    self.convert_notebooks()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 457, in convert_notebooks\n",
      "    self.convert_single_notebook(notebook_filename)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 428, in convert_single_notebook\n",
      "    output, resources = self.export_single_notebook(notebook_filename, resources, input_buffer=input_buffer)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 357, in export_single_notebook\n",
      "    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 165, in from_filename\n",
      "    return self.from_file(f, resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 183, in from_file\n",
      "    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\__init__.py\", line 141, in read\n",
      "    return reads(fp.read(), as_version, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\__init__.py\", line 74, in reads\n",
      "    nb = reader.reads(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 58, in reads\n",
      "    nb_dict = parse_json(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 17, in parse_json\n",
      "    raise NotJSONError((\"Notebook does not appear to be JSON: %r\" % s)[:77] + \"...\")\n",
      "nbformat.reader.NotJSONError: Notebook does not appear to be JSON: '{\\n \"cells\": [\\n  {\\n   \"cell_type\": \"c...\n",
      "[NbConvertApp] Converting notebook Euler12.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 2480 bytes to Euler12.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler13.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 904 bytes to Euler13.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler14.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] ERROR | Timeout waiting for execute reply (30s).\n",
      "                If your cell should take longer than this, you can increase the timeout with:\n",
      "\n",
      "                    c.ExecutePreprocessor.timeout = SECONDS\n",
      "\n",
      "                in jupyter_nbconvert_config.py\n",
      "                \n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 189, in run_cell\n",
      "    msg = self.kc.shell_channel.get_msg(timeout=timeout)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\jupyter_client\\blocking\\channels.py\", line 57, in get_msg\n",
      "    raise Empty\n",
      "queue.Empty\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\Scripts\\jupyter-nbconvert-script.py\", line 5, in <module>\n",
      "    main()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\jupyter_core\\application.py\", line 267, in launch_instance\n",
      "    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\traitlets\\config\\application.py\", line 596, in launch_instance\n",
      "    app.start()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 293, in start\n",
      "    self.convert_notebooks()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 457, in convert_notebooks\n",
      "    self.convert_single_notebook(notebook_filename)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 428, in convert_single_notebook\n",
      "    output, resources = self.export_single_notebook(notebook_filename, resources, input_buffer=input_buffer)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 357, in export_single_notebook\n",
      "    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 165, in from_filename\n",
      "    return self.from_file(f, resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 183, in from_file\n",
      "    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\notebook.py\", line 26, in from_notebook_node\n",
      "    nb_copy, resources = super(NotebookExporter, self).from_notebook_node(nb, resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 130, in from_notebook_node\n",
      "    nb_copy, resources = self._preprocess(nb_copy, resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 302, in _preprocess\n",
      "    nbc, resc = preprocessor(nbc, resc)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\base.py\", line 47, in __call__\n",
      "    return self.preprocess(nb,resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 145, in preprocess\n",
      "    nb, resources = super(ExecutePreprocessor, self).preprocess(nb, resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\base.py\", line 70, in preprocess\n",
      "    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 161, in preprocess_cell\n",
      "    outputs = self.run_cell(cell)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\preprocessors\\execute.py\", line 207, in run_cell\n",
      "    raise exception(\"Cell execution timed out, see log\"\n",
      "TimeoutError: Cell execution timed out, see log for details.\n",
      "[NbConvertApp] Converting notebook Euler15.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 1352 bytes to Euler15.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler16.ipynb to notebook\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 14, in parse_json\n",
      "    nb_dict = json.loads(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\__init__.py\", line 319, in loads\n",
      "    return _default_decoder.decode(s)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\decoder.py\", line 339, in decode\n",
      "    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\decoder.py\", line 355, in raw_decode\n",
      "    obj, end = self.scan_once(s, idx)\n",
      "json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 16 column 1 (char 283)\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\Scripts\\jupyter-nbconvert-script.py\", line 5, in <module>\n",
      "    main()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\jupyter_core\\application.py\", line 267, in launch_instance\n",
      "    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\traitlets\\config\\application.py\", line 596, in launch_instance\n",
      "    app.start()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 293, in start\n",
      "    self.convert_notebooks()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 457, in convert_notebooks\n",
      "    self.convert_single_notebook(notebook_filename)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 428, in convert_single_notebook\n",
      "    output, resources = self.export_single_notebook(notebook_filename, resources, input_buffer=input_buffer)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 357, in export_single_notebook\n",
      "    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 165, in from_filename\n",
      "    return self.from_file(f, resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 183, in from_file\n",
      "    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\__init__.py\", line 141, in read\n",
      "    return reads(fp.read(), as_version, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\__init__.py\", line 74, in reads\n",
      "    nb = reader.reads(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 58, in reads\n",
      "    nb_dict = parse_json(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 17, in parse_json\n",
      "    raise NotJSONError((\"Notebook does not appear to be JSON: %r\" % s)[:77] + \"...\")\n",
      "nbformat.reader.NotJSONError: Notebook does not appear to be JSON: '{\\n \"cells\": [\\n  {\\n   \"cell_type\": \"m...\n",
      "[NbConvertApp] Converting notebook Euler2.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 1055 bytes to Euler2.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler20.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 1241 bytes to Euler20.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler3.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 3715 bytes to Euler3.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler4.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 1129 bytes to Euler4.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler5.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 1533 bytes to Euler5.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler6.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 2650 bytes to Euler6.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler7.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 219531 bytes to Euler7.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler8.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 4053 bytes to Euler8.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler81.ipynb to notebook\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 14, in parse_json\n",
      "    nb_dict = json.loads(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\__init__.py\", line 319, in loads\n",
      "    return _default_decoder.decode(s)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\decoder.py\", line 339, in decode\n",
      "    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\json\\decoder.py\", line 355, in raw_decode\n",
      "    obj, end = self.scan_once(s, idx)\n",
      "json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 29 column 1 (char 462)\n",
      "\n",
      "During handling of the above exception, another exception occurred:\n",
      "\n",
      "Traceback (most recent call last):\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\Scripts\\jupyter-nbconvert-script.py\", line 5, in <module>\n",
      "    main()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\jupyter_core\\application.py\", line 267, in launch_instance\n",
      "    return super(JupyterApp, cls).launch_instance(argv=argv, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\traitlets\\config\\application.py\", line 596, in launch_instance\n",
      "    app.start()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 293, in start\n",
      "    self.convert_notebooks()\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 457, in convert_notebooks\n",
      "    self.convert_single_notebook(notebook_filename)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 428, in convert_single_notebook\n",
      "    output, resources = self.export_single_notebook(notebook_filename, resources, input_buffer=input_buffer)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\nbconvertapp.py\", line 357, in export_single_notebook\n",
      "    output, resources = self.exporter.from_filename(notebook_filename, resources=resources)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 165, in from_filename\n",
      "    return self.from_file(f, resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbconvert\\exporters\\exporter.py\", line 183, in from_file\n",
      "    return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\__init__.py\", line 141, in read\n",
      "    return reads(fp.read(), as_version, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\__init__.py\", line 74, in reads\n",
      "    nb = reader.reads(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 58, in reads\n",
      "    nb_dict = parse_json(s, **kwargs)\n",
      "  File \"C:\\Users\\mkcarra\\Anaconda3\\lib\\site-packages\\nbformat\\reader.py\", line 17, in parse_json\n",
      "    raise NotJSONError((\"Notebook does not appear to be JSON: %r\" % s)[:77] + \"...\")\n",
      "nbformat.reader.NotJSONError: Notebook does not appear to be JSON: '{\\n \"cells\": [\\n  {\\n   \"cell_type\": \"c...\n",
      "[NbConvertApp] Converting notebook Euler9.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 1578 bytes to Euler9.nbconvert.ipynb\n",
      "[NbConvertApp] Converting notebook Euler96.ipynb to notebook\n",
      "[NbConvertApp] Executing notebook with kernel: python3\n",
      "[NbConvertApp] Writing 17744 bytes to Euler96.nbconvert.ipynb\n"
     ]
    }
   ],
   "source": [
    "for filename in filelist:\n",
    "    !jupyter nbconvert $filename --to notebook --execute"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "!del Euler*.nbconvert.ipynb"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.1"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
